streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBVersionedStoreSegmentValueFormatter.java

```diff
@@ -136,6 +137,9 @@ static SegmentValue newSegmentValueWithRecord(
     }
 
     interface SegmentValue {
+        long getMinTimestamp();
+
+        long getNextTimestamp();
```

Comment:
Why do we need to add this here? The newly added `ReadOnlyPartiallyDeserializedSegementValue` does int implement his interface.